### PR TITLE
usm docs fixes

### PR DESCRIPTION
**Pull request by guyarb**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/universal_service_monitoring/setup.md [hunk 1]
```diff
@@ -764,7 +767,37 @@ DD_SYSTEM_PROBE_PROCESS_SERVICE_INFERENCE_ENABLED=true
 {{% /tab %}}
 
 {{< /tabs >}}
-{{< /collapse-content >}} 
+{{< /collapse-content >}}
+
+{{< collapse-content title="Non-IIS service discovery on Windows" level="h4" >}}
+<div class="alert alert-info">
+Universal Service Monitoring is available in <strong>beta</strong> to monitor non IIS services running on a Windows machines.
```
Comment (aliciascott) [inline content/en/universal_service_monitoring/setup.md:774]:
```suggestion
Universal Service Monitoring is available in <strong>beta</strong> to monitor non IIS services running on Windows environments.
```

File: content/en/universal_service_monitoring/setup.md [hunk 2]
```diff
@@ -764,7 +767,37 @@ DD_SYSTEM_PROBE_PROCESS_SERVICE_INFERENCE_ENABLED=true
...
+
+{{< tabs >}}
+{{% tab "Configuration file" %}}
+
+Add the following configuration to the `system-probe.yaml`:
+
+```yaml
+service_monitoring_config:
+  enabled: true
+  process_service_inference:
```
Comment (aliciascott) [inline content/en/universal_service_monitoring/setup.md:787]:
```suggestion
process_service_inference:
```
I this supposed to be indented ?

File: content/en/universal_service_monitoring/setup.md [hunk 3]
```diff
@@ -764,7 +767,37 @@ DD_SYSTEM_PROBE_PROCESS_SERVICE_INFERENCE_ENABLED=true
...
+    enabled: true
```
Comment (aliciascott) [inline content/en/universal_service_monitoring/setup.md:788]:
```suggestion
  enabled: true
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @guyarb just a minor suggestion and then I had a question about the indenting in the yaml file, let me know when fixed and i'll review and get it merged

File: content/en/universal_service_monitoring/setup.md (hunk 2 above)
Comment (guyarb) [reply content/en/universal_service_monitoring/setup.md:787]: yes, it is nested on intention

File: content/en/universal_service_monitoring/setup.md (hunk 3 above)
Comment (guyarb) [reply content/en/universal_service_monitoring/setup.md:788]: same as above

**Pull request merged**